frontend/src/components/CourseCartCard/CourseCartCard.tsx

```diff
@@ -17,6 +18,17 @@ type Props = {
 const CourseCartCard = ({ code, title }: Props) => {
   const dispatch = useDispatch();
   const navigate = useNavigate();
+  const { token } = useSelector((state: RootState) => state.settings);
+
+  const handleConfirm = () => {
+    try {
+      axios
+        .post(`/planner/removeCourse`, JSON.stringify({ courseCode: code }), { params: { token } });
+    } catch (e) {
+      // esling-disable-next-line no-console
+      console.error('Error at removeCourse', e);
+    }
+  };
```

Comment:
async await plzz

Reply:
also in instead of logging the error you can show a pop up notification saying that there was an error so the user knows that the course didn't get removed properly